### PR TITLE
Retrieve all items

### PR DESCRIPTION
**Pull request by TildaYngman**
Created a function to fetch all items of a given category from the db. Added an API method to handle the fetch function. Corrected an issue with the db connection configuration. Called the item fetch function in Adventures and Culture pages.

### CODE REVIEW

File: client/src/scripts/api.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+//function calls get method
+
```
Comment (felix-odenhall) [inline client/src/scripts/api.js]: Forgot to remove pseudo code

File: server/index.js [hunk 2]
```diff
@@ -31,14 +32,20 @@ const requestLogger = (request, response, next) => {
 
 app.use(requestLogger);
 app.use(express.json());
-app.use(cors({ origin: "http://localhost:3000" }));
```
Comment (felix-odenhall) [inline server/index.js:34]: Why was this changed?

Comment (BenCiantar) [reply server/index.js:34]: This was an attempted bug fix during testing that was overlooked. I'll revert it now, thanks.

**Review by ElinaHulbert — APPROVED** [review]
